pkgs/development/compilers/cudatoolkit/redist/find_outputs/find_outputs/manifest.py

```diff
@@ -0,0 +1,38 @@
+import json
+import logging
+from pathlib import Path
+from typing import (
+    Any,
+    TypeAlias,
+)
+
+from pydantic import BaseModel
+from pydantic.tools import parse_obj_as
+
+from find_outputs import generic
+
+
+class Release(BaseModel):
+    relative_path: str
+    sha256: str
+    md5: str
+    size: str
+
+
+Package: TypeAlias = generic.Package[Release]
+
+URL_PREFIX = "https://developer.download.nvidia.com/compute/cuda/redist"
```

Comment:
AFAICT this is only referenced once, might as well move it there

Reply:
https://github.com/ConnorBaker/cuda-redist-find-features/commit/31ba5f591c8938995126cd27d5581b2c1fbf7234